views/tables/ChartsTable/index.jsx

```diff
@@ -48,7 +45,18 @@ const ChartsTable = ({
   const cellRenderer = (chart, property) => {
     switch (property) {
       case 'owner':
-        return chart['chartOwner'].display_name
+        return (
+          <div>
+            <Avatar
+              alt={chart['chartOwner'].display_name[0]}
+              src={chart['chartOwner'].icon}
+              sx={{ width: 24, height: 24, float: 'left' }}
```

Comment:
we shouldn't have to use floats now that we have tools like flexbox and css grid.

Reply:
[c5452ad](https://github.com/AMP-SCZ/dpdash/pull/530/commits/c5452adac507cb21aef66b1d754d6d34b1f7e517)